cstar/roms/simulation.py

```diff
@@ -1428,27 +1414,17 @@ def run(
         # these are conceptually different:
         run_path = output_dir
```

Comment:
they might be conceptually different but they have the same value.... is this comment still valid? if they're equal, how are they conceptually different? Can we update or remove this while we're here